gadget-local.Dockerfile

```diff
@@ -4,7 +4,7 @@
 # https://github.com/kinvolk/bcc/commit/4e8e8c65335d650414cd05a38b6860080dba81da
 # See BCC section in docs/CONTRIBUTING.md for further details.
 
-FROM docker.io/kinvolk/bcc:202107061407494e8e8c
+FROM quay.io/kinvolk/bcc:4e8e8c65335d650414cd05a38b6860080dba81da-focal-release
 
 RUN set -ex; \
 	export DEBIAN_FRONTEND=noninteractive; \
```

Comment:
Can you remove the file directly in this commit? It seems a bit weird to modify the file in one commit and delete it in the next one.

Reply:
Ok, I'll squash the two commits